index.js

```diff
@@ -1,53 +1,131 @@
 'use strict'
 
+const { randomBytes } = require('crypto')
 const fp = require('fastify-plugin')
 const helmet = require('helmet')
-const crypto = require('crypto')
 
-module.exports = fp(function (app, options, next) {
-  const enableCSPNonces = options.enableCSPNonces
-  // clear options as helmet will throw when any options is "true"
-  options.enableCSPNonces = undefined
+function helmetPlugin (fastify, options, next) {
+  // helmet will throw when any options is explicitly set to "true"
```

Comment:
```suggestion
  // helmet will throw when any option is explicitly set to "true"
```

Reply:
Applied all your suggestions.
Thanks a lot for taking the time to review this PR ^^